l-sgov/l-sgov-111-2009-slovník.ttl

```diff
@@ -5,11 +5,15 @@
 @prefix xsd:   <http://www.w3.org/2001/XMLSchema#> .
 @prefix rdfs:  <http://www.w3.org/2000/01/rdf-schema#> .
 
-l-sgov-111-2009:slovník
-        a               <https://slovník.gov.cz/veřejný-sektor/pojem/slovník> , owl:Ontology ;
-        rdfs:label      "Vocabulary of the Act No. 111/2009 Col., about basic registers - vocabulary"@en , "Slovník zákona č. 111/2009 Sb., o základních registrech - slovník"@cs ;
+<https://slovník.gov.cz/legislativní/sbírka/111/2009>
```

Comment:
1. Vocabulary was supposed to have dc:title instead of rdfs:label (AFAIK, this is how it comes out of Modelio already)
2. Vocabulary IRI was supposed to **NOT** end with /slovník

Reply:
this shall be part of the https://docs.google.com/document/d/1daoJKZEGbanRfh6ub2Fz8pwEU1dlUnjQHPJ6knt_EKg/edit#

Reply:
anyway 
1. vocabulary has both dc:title and rdfs:label (which is written in the document)
2. vocabulary IRI does **NOT** end with /slovník, it is <https://slovník.gov.cz/legislativní/sbírka/111/2009>

Reply:
>     1. Vocabulary was supposed to have dc:title instead of rdfs:label (AFAIK, this is how it comes out of Modelio already)
vocabulary has both dc:title and rdfs:label (which is written in the document), at least how I understood that,
> 
>     2. Vocabulary IRI was supposed to **NOT** end with /slovník

vocabulary IRI does **NOT** end with /slovník, it is <https://slovník.gov.cz/legislativní/sbírka/111/2009>

Reply:
> As per https://lists.w3.org/Archives/Public/public-gld-wg/2013Mar/0109.html (with which I agree) we should be consistent to use either:
> 
>     1. rdfs:label, rdfs:comment
> 
>     2. rdfs:label, rdfs:comment, dc:title, dc:description
> 
>     3. dc:title,dc:description + include (dc:title subPropertyOf rdfs:label) and (dc:description subPropertyOf rdfs:comment).
> 
> 
> I am for variant 3).

I did not get what was the purpose of the document https://docs.google.com/document/d/1daoJKZEGbanRfh6ub2Fz8pwEU1dlUnjQHPJ6knt_EKg/edit# then.

Reply:
1. Could you point out to me in the document (in the "Final proposal" part) where it is written that a vocabulary has a rdfs:label?

2. `l-sgov-111-2009:slovník` does seem to me to end with /slovník when you expand the namespace...

Reply:
>     1. Could you point out to me in the document (in the "Final proposal" part) where it is written that a vocabulary has a rdfs:label?

sorry, I misunderstood. So the goal is to have ONLY those in the document. Mapping then shall be rdfs:label -> dc:title and rdfs:comment -> dc:description?
> 
>     2. `l-sgov-111-2009:slovník` does seem to me to end with /slovník when you expand the namespace...

yes, but it is in deleted part in diff, replaced by <https://slovník.gov.cz/legislativní/sbírka/111/2009>

Reply:
in the next PR is rdfs:label changed to dc:title

Reply:
1. Yes, basically, as it was originally:) But now, the same will be used by TermIt as well.
2. You are absolutely right, I didn't notice those lines were marked as deleted in the diff. Sorry about that.

Reply:
I'm resolving this conversation then.